src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/Microsoft.CodeAnalysis.Razor.Compiler.csproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Razor is a markup syntax for adding server-side logic to web pages. This package contains the Razor compiler.</Description>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Think we will need to multi-target `net8.0` and `netsandard2.0` here. The former will become a requirement from source build at some point. May as well lead with it.

Reply:
Added here, not to the transport package, I assume it's not needed there.